runtime/src/bank.rs

```diff
@@ -3693,14 +3693,18 @@ impl Bank {
     }
 
     #[cfg(test)]
-    fn add_account_and_update_capitalization(&self, pubkey: &Pubkey, new_account: &Account) {
+    fn store_account_and_update_capitalization(&self, pubkey: &Pubkey, new_account: &Account) {
```

Comment:
Well, I renamed this to sound like a cousin to `store_account`. Introducing different verb `add_` was a bad idea...